react/legacy/components/ProductQuantityStepper.js

```diff
@@ -6,21 +6,21 @@ import { Pixel } from 'vtex.pixel-manager/PixelContext'
 import { debounce } from 'debounce'
 import { injectIntl, intlShape } from 'react-intl'
 import { compose, graphql } from 'react-apollo'
-import gql from 'graphql-tag'
+import { parse } from 'graphql'
 
 import { productShape } from '../../utils/propTypes'
 
-const UPDATE_ITEMS_MUTATION = gql`
+const UPDATE_ITEMS_MUTATION = parse(`
   mutation updateItems($items: [MinicartItem]) {
     updateItems(items: $items) @client
   }
-`
+`)
 
-const UPDATE_LOCAL_ITEMS_MUTATION = gql`
+const UPDATE_LOCAL_ITEMS_MUTATION = parse(`
   mutation updateLocalItems($items: [MinicartItem]) {
     updateLocalItems(items: $items) @client
   }
-`
+`)
```

Comment:
Isnt it better to create a .graphql file instead?

Reply:
No, this would generate a persisted query. Generating a persisted query means the `__RUNTIME__.cacheHints` will be populated with these two mutations, increasing the size of this object. This would be necessary if these queries get to graphql-server. Since these mutations are client only, we should not import these queries, thus decreasing the overall store size